src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/prefermethodreference/InputPreferMethodReferenceMethodCalls.java

```diff
@@ -0,0 +1,80 @@
+package com.puppycrawl.tools.checkstyle.checks.coding.prefermethodreference;
+
+import static java.lang.Long.sum;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;import java.util.Optional;import java.util.concurrent.Callable;
```

Comment:
Why these imports are messed up? Imports have no effect to this check.
If this is in order not to alter a large number of asserts, you can move the first two cases to the end - the rest will remain in place.

Reply:
just garbage, deleted.